uvloop/handles/stream.pyx

```diff
@@ -420,6 +420,7 @@ cdef class UVStream(UVBaseTransport):
         self._buffer_size += dlen
         self._buffer.append(data)
 
+    cdef inline _write(self):
```

Comment:
I think we should we rename it to `_queue_write()`?

Reply:
Good point! Though I would call it `_start_write()`, because:

1. This method may try to write immediately without queuing.
2. There is also `loop._queue_write()` which actually schedules a write in the next iteration.
3. In `UVStream`, we also have `_exec_write()`, `_try_write()`, now with `_buffer_write()`, `_queue_write()` just confuses people a lot - which does what?

So I'm also adding some comments now.

Reply:
Awesome, this makes sense! Maybe `_initiate_write()`? Like it's closer to "start now or soon".

Reply:
Sure thing!